acto/lib/test_operator_config.py

```diff
@@ -0,0 +1,15 @@
+import glob
+import json
+import os
+
+import pytest
+
+from acto.lib.operator_config import OperatorConfig
+
+data_dir = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))), "data")
+
+
+@pytest.mark.parametrize("path", glob.glob(os.path.join(data_dir, '**', 'config.json')))
+def test_operator_config(path):
+    """Check if all teh operator configs can be loaded"""
```

Comment:
We currently haven't set a standard in terms of using single quotes and double quotes in the project. People typically pick one and stick to it throughout the project. Since the rest of the project has been using single quotes so far, could you also use the single quotes for now. We can change all of the quotes if we decide otherwise later.

Reply:
NVM, I think the convention for docstring is to always use triple double quotes. We would need to update the other docstrings all together in the project at some point.